app/helpers/link_helper.rb

```diff
@@ -0,0 +1,23 @@
+module LinkHelper
+
+  def link_to_new_group_ling ling, depth, options={}, &block
```

Comment:
this method changes the link, if the user is an expert, he can't perform some kind of actions